Makefile

```diff
@@ -119,8 +119,10 @@ build-java-no-tests:
 # Go SDK
 
 install-go-ci-dependencies:
-	go get -u github.com/golang/protobuf/protoc-gen-go
-	go get -u golang.org/x/lint/golint
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
```

Comment:
We can't let this dependency float. We should pin it to a specific version I think.